### ISSUE TITLE
Remove global lambdaLayer from warmup lambdas

### ISSUE THREAD

**Issue by swirkens** [juanjoDiaz/serverless-plugin-warmup#290]
Hi,
I have globally set lambda layer for all my lambdas. Is there any way to detach it from warmup lambda? Can't see this option in config.

serverless.yml
```
plugins:
  - serverless-webpack
  - serverless-latest-layer-version
  - serverless-plugin-warmup

provider:
  name: aws
...
  layers:
    - arn:aws:lambda:${self:provider.region}:${AWS::AccountId}:layer:${self:custom.lambdaLayerName}:latest

functions:
  ...
```

**Comment by juanjoDiaz** [comment]
Hi @swirkens,

As far as I know there is no way to remove layers at a function level if you set it up at the provider level. Or is it?

**Comment by swirkens** [comment]
There is a way to do that. You can set layers to empty array at function level ;)

```
functions:
  example:
    layers: []
```

**Comment by juanjoDiaz** [comment]
Resolved in v6.1.0
